modules/kubernetes/grafana-agent/charts/grafana-agent-extras/templates/metricsinstance.yaml

```diff
@@ -16,20 +16,23 @@ spec:
           name: {{ .Values.credentialsSecretName }}
           key: metrics_password
   serviceMonitorNamespaceSelector:
-    matchLabels:
-      xkf.xenit.io/kind: tenant
+    matchExpressions:
+      - key: "xkf.xenit.io/kind"
+        operator: "Exists"
   serviceMonitorSelector:
     matchLabels:
       xkf.xenit.io/monitoring: tenant
   podMonitorNamespaceSelector:
-    matchLabels:
-      xkf.xenit.io/kind: tenant
+    matchExpressions:
+      - key: "xkf.xenit.io/kind"
+        operator: "Exists"
```

Comment:
How would this be a problem. This enables tenant service monitors in platform namespaces. Not the other way around.

Reply:
@NissesSenap if a tenant adds a serviceMonitor with platform, our prometheus won't pick it up since it has a namespace selector with it. This change introduces the ability for us to provide the grafana-agent (tenant monitor) with additional metrics / logs - but not the other way around.

Reply:
I mean the other way around. They can by mistake scrape platform metrics.
I'm not worried about the platform metrics.

Reply:
How?